content/applications/marketing/events/event_essentials.rst

```diff
@@ -2,283 +2,384 @@
+event-planning in the future.
 
-To create a new stage, click :guilabel:`Add a Column` and provide a befitting title to reflect the
-purpose of that stage.
+However, before a user can take advantage of all the detailed tools this application provides, they
+must first familiarize themselves with the basic dashboards, settings, and features, which is what
+this event essentials documentation covers below.
 
-To reorganize the stages, just drag-and-drop them until they are in the correct order. Stages can
-also be “folded” in the kanban for a cleaner presentation, via the :guilabel:`Settings` menu to the
-right of each stage's title, which is represented by a gear icon.
+.. _event_essentials/dashboards:
+
+Dashboards
+==========
+
+When the *Events* application is opened, Odoo reveals the main :guilabel:`Events` dashboard, which
+can be viewed in a number of different ways. Those different view options are accessible from the
+:guilabel:`Events` dashboard in the upper-right corner, via a series of view-related icon buttons.
+
+.. image:: event_essentials/view-icons.png
+   :align: center
+   :alt: The various view icon buttons in the upper-right corner of the dashboard.
+
+The different view options are:
+
+- :ref:`Kanban <event_essentials/dashboards/kanban>` (default)
+- :ref:`Calendar <event_essentials/dashboards/calendar>`
+- :ref:`List <event_essentials/dashboards/list>`
+- :ref:`Gantt <event_essentials/dashboards/gantt>`
+- :ref:`Pivot <event_essentials/dashboards/pivot>`
+- :ref:`Graph <event_essentials/dashboards/graph>`
+- :ref:`Map <event_essentials/dashboards/map>`
+- :ref:`Activity <event_essentials/dashboards/activity>`
+
+If at any time the user desires to navigate back to the main dashboard, they can do so by going to
+:menuselection:`Events app --> Events`.
+
+By default, the :guilabel:`Events` dashboard shows *all* the events in the database.
+
+.. _event_essentials/dashboards/kanban:
+
+Kanban view
+-----------
+
+The Kanban view is the default view of the main :guilabel:`Events` dashboard, and is represented by
+the view option displayed as the :guilabel:`two downward-facing columns` icon, located in the
+upper-right corner.
+
+This view showcases all the events in the database in their respective stages. By default, the
+stages are: :guilabel:`New`, :guilabel:`Booked`, :guilabel:`Announced`, :guilabel:`Ended`, and
+:guilabel:`Cancelled`.
+
+.. image:: event_essentials/kanban-dashboard.png
+   :align: center
+   :alt: The default Kanban dashboard of the Odoo Events application.
 
 .. note::
-   The gear icon, by default, is hidden and will appear next to the  :guilabel:`+` icon when moused
-   over.
+   The :guilabel:`Ended` and :guilabel:`Cancelled` stages are folded, by default, and located to the
+   right of the other stages.
 
-When clicked, a drop-down menu of settings will appear.
+To add a new stage to the Kanban pipeline, click the :guilabel:`+ Stage` button, located to the
```

Comment:
in this instance, since the plus sign on the UI isn't as pronounced as it is in other areas of the platform, I'm opting for the typed "+" sign here. One, that ensures that it shows up for every reader. Two, it more accurately represents that button, in my opinion. And three, it keeps that GUI clean, whereas otherwise, I'd have to add a (plus sign) mention within the GUI. 👍